reports/dcim-reports/CheckCableLocality.py

```diff
@@ -0,0 +1,32 @@
+from extras.reports import Report
+from dcim.models import Cable, RearPort
+from dcim.constants import *
+from circuits.models import CircuitTermination
+
+CABLE_TYPES_OK_BETWEEN_RACKS = {
+    CABLE_TYPE_DAC_PASSIVE,
+}
+
+class CheckCableLocality(Report):
+    description = "Warn on cables between racks, error on cables between sites"
+
+    def test_cable_endpoints(self):
+        for cable in Cable.objects.prefetch_related('termination_a','termination_b').all():
+            if isinstance(cable.termination_a, CircuitTermination) or isinstance(cable.termination_b, CircuitTermination):
+                continue
+            if cable.termination_a.device.site != cable.termination_b.device.site:
```

Comment:
You're not checking for `PowerFeed`s.
This will fail if a cable has a `PowerFeed` as one of the sides.
Add: 
```
from dcim.models import PowerFeed
if isinstance(cable.termination_a, PowerFeed) or isinstance(cable.termination_b, PowerFeed):
    continue
```

Reply:
Thanks, now fixed in a more general way